src/examples/cicd-eb-docker-service.yml

```diff
@@ -17,7 +17,7 @@ usage:
             executor-class: medium
 
         - cyber4all/lint-app:
-            docker-image: cimg/node
+            docker-image: cimg/node:18.0.0
```

Comment:
I would not recommend pointing to latest. LTS for cimg/node is 16.15.0

Reply:
This works